src/render.rs

```diff
@@ -102,7 +102,8 @@ impl<'a> Renderer<'a> {
 
         // dummy statement to satisfy the compiler
         // TODO: make it so that's not needed
-        self.context.lookup(key).cloned().ok_or_else(|| FieldNotFound(key.to_string()))
+        self.context.pointer(&("/".to_string() + &key.replace(".", "/"))).cloned()
```

Comment:
Slice patterns don't seem stable yet and it still seems a bit uncertain looking at https://github.com/rust-lang/rust/issues/23121